CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # v2.2.0
 ### Add
-- Add [Step List component](https://github.com/AdactiveSAS/adsum-react-components/tree/master/packages/adsum-steplist).
+- Add [Step List component](https://github.com//adsum-react-components/tree/master/packages/adsum-steplist).
```

Comment:
Don't you need "adactiveasia" between de two " / " ?

Reply:
should be adactiveasia between two "/"